examples/Training/java/src/training/HowToCreateMapAnnotation.java

```diff
@@ -0,0 +1,134 @@
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ *------------------------------------------------------------------------------
+ */
+package training;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import omero.gateway.Gateway;
+import omero.gateway.LoginCredentials;
+import omero.gateway.SecurityContext;
+import omero.log.SimpleLogger;
+import omero.model.ImageI;
+import omero.model.NamedValue;
+import omero.gateway.facility.BrowseFacility;
+import omero.gateway.facility.DataManagerFacility;
+import omero.gateway.facility.TablesFacility;
+import omero.gateway.model.ExperimenterData;
+import omero.gateway.model.FileAnnotationData;
+import omero.gateway.model.ImageData;
+import omero.gateway.model.MapAnnotationData;
+
+
+
+/**
+ * Creates a Map Annotation and links it to a given image.
+ * @author Jean-Marie Burel &nbsp;&nbsp;&nbsp;&nbsp;
+ * <a href="mailto:j.burel@dundee.ac.uk">j.burel@dundee.ac.uk</a>
+ * @since 5.4
+ */
+public class HowToCreateMapAnnotation {
+
+    /** The server address.*/
+    private static String hostName = "serverName";
+
+    /** The username.*/
+    private static String userName = "userName";
+
+    /** The password.*/
+    private static String password = "password";
+
+    private Gateway gateway;
+
+    private SecurityContext ctx;
+
+    /**
+     * start-code
+     */
+
+ // Create MapAnnotation
+ // ====================
+
+    /** 
+     * Creates a table and links to an Image.
+     * @param imageID The omero image identifier
+     * @throws Exception
+     */
+    private void createMapAnnotationandLinkToImage(long imageID)
+            throws Exception
+    {
+        List<NamedValue> result = new ArrayList<NamedValue>();
+        result.add(new NamedValue("mitomycin-A", "20mM"));
+        result.add(new NamedValue("PBS", "10mM"));
+        result.add(new NamedValue("incubation", "5min"));
+        result.add(new NamedValue("temperature", "37"));
+        result.add(new NamedValue("Organism", "Homo sapiens"));
+        MapAnnotationData data = new MapAnnotationData();
+        data.setContent(result);
+        data.setDescription("Training Example");
+        //Use the following namespace if you want the annotation to be displayed
+        //in the right-hand panel
+        data.setNameSpace(MapAnnotationData.NS_CLIENT_CREATED);
```

Comment:
I guess that's the same ns as ```omero.constants.metadata.NSCLIENTMAPANNOTATION```?
The doc-string should say "...want the annotation to be *editable* in the webclient & Insight"
(All map-anns are displayed).

Reply:
it is the same. I will update the doc, that's what I meant 
thanks for checking